src/ansys/pre_commit_hooks/add_license_headers.py

```diff
@@ -89,6 +91,41 @@ def set_lint_args(parser):
     return parser.parse_args()
 
 
+def copy_assets(proj_root, args):
```

Comment:
You will copy them and leave them there?

Why not deleting them after the hook has ran so that these files are not committed to the repository?

Reply:
I think we still need the folders to be in their repositories because each repo may have different dep5 file content since it has to match the layout of the repository. Also, if someone is using a custom jinja2 file or license then they would have to re-make it every time the hook runs. 

Whenever they first run the hook, they'll have to commit the reuse and LICENSES folders. Otherwise, the following runs should only have to commit the ansys.jinja2 file and/or MIT.txt file if there is a change to the content of those files